content/en/docs/tasks/tools/install-kubeadm.md

```diff
@@ -249,13 +249,10 @@ flag you need to change is `--cgroup-driver`. If it's already set, you can updat
 sed -i "s/cgroup-driver=systemd/cgroup-driver=cgroupfs/g" /etc/systemd/system/kubelet.service.d/10-kubeadm.conf
 ```
 
-Otherwise, you will need to open the systemd file and add the flag to an existing environment line.
-
-Then restart kubelet:
+Otherwise, you will need to open the systemd file and add the flag as below.
 
 ```bash
-systemctl daemon-reload
-systemctl restart kubelet
+Environment="KUBELET_CGROUP_ARGS=--cgroup-driver=cgroupfs"
```

Comment:
^ a couple of problems with the above:
- `cgroupsfs` is already the default value so this argument would do nothing.
- the removal of this, is not advised as it instructs the user to restart the kubelet:
```
systemctl daemon-reload
systemctl restart kubelet
```
we are moving towards a better way of handling this in 1.11, so i'm going to close this PR.

thank you for your contribution, @akaey. feel free to send more changes, but first sing the CLA.